src/components/plugins/SchemaToHtml.vue

```diff
@@ -0,0 +1,281 @@
+<template>
+    <div class="bd-markdown">
+        <div class="doc-alert alert alert-warning" role="alert" v-if="page.deprecated">
+            <p>⚠ Deprecated</p>
+        </div>
+
+        <SchemaToCode language="yaml" :code='`type: "${getPageName()}"`'/>
+        <p v-if="page.title">
+            <span style="font-size:1.5em;" v-html="replaceText(page.title)"/>
+        </p>
+        <div v-if="page.description" v-html="generateDescHtml"/>
+        <h2 id="examples" v-if="page.body.children['examples']">
+            <a href="#examples">Examples</a>
+        </h2>
+        <template v-for="example in page.body.children['examples']"
+                  v-if="page.body.children['examples']">
+            <blockquote class="blockquote">
+                <p v-html="replaceText(example.title)" />
+            </blockquote>
+            <SchemaToCode
+              language="yaml"
+              :code="example.title?.split('```yaml')[1]?.split('```')[0]"
+              v-if="example.title?.split('```yaml')[1]?.split('```')[0]"
+            />
+            <SchemaToCode :language="example.lang" :code="example.code" v-if="example.code"/>
+        </template>
+        <template v-for="(pageBlock, key) in page.body.children" v-if="page.body.children">
+            <template v-if="key !== 'examples'">
```

Comment:
Please add this extra logic to examples so that we have an id & type property embed in them
![image](https://github.com/kestra-io/ui-libs/assets/37618489/9933c7c3-997a-4fc4-9c9e-8187f6a5ec2b)

![image](https://github.com/kestra-io/ui-libs/assets/37618489/876eb939-80fd-4062-96de-b15688ff2a15)

Reply:
fixed